src/index.html

```diff
@@ -1,11 +1,11 @@
-﻿﻿﻿<!DOCTYPE HTML>
+﻿﻿<!DOCTYPE HTML>
```

Comment:
I have no idea -- using my diff tool I don't see a difference.. and if the illegal character were there it wouldn't pass linting..